src/components/Filter.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const Filter = (props) => {
+    const records = props.moviecount;
```

Comment:
Hint: `moviecount` can be extracted in arguments part of the function like: `const Filter = ({ moviecount }) => <div>...</div>`